### PR TITLE
Kinfitter available from python

### PR DESCRIPTION
**Pull request by HeinerTholen**
Adding ClassDef's, classes.h and classes_def.xml in order to make TKinFitter and related classes available from python. This is practical for development, e.g. with jupyter notebooks.

### CODE REVIEW

**Comment by fabiocos** [comment]
please test

File: PhysicsTools/KinFitter/src/classes.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//     TFitParticleEtThetaPhi fpetp;
+//     TFitParticleMCCart fpmcc;
+//     TFitParticleMCMomDev fmmccd;
+//     TFitParticleMCPInvSpher fpmcpis;
+//     TFitParticleMCSpher fpmcs;
+//     TFitParticleMomDev fpmd;
+//     TFitParticleSpher fps;
+//     TKinFitter kf;
+//     TSLToyGen sltg;
+
```
[the remark this answers is not in the document]
Comment (HeinerTholen) [reply PhysicsTools/KinFitter/src/classes.h]: sorry, I uncommented the classes when I was looking to fix an issue. They are added back in now. They all produce a warning, but everything compiles fine:  

    warning: 'IsA' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Pull request merged**